assets/TenancyServiceProvider.stub.php

```diff
@@ -118,7 +118,20 @@ public function events()
 
     public function register()
     {
-        //
+        /**
```

Comment:
1. I'd move this to a separate method and call that from here
2. Can we use `boot()` instead?

Reply:
Good idea with the separate method. Calling the method in `boot()` seems to be working fine

Reply:
I updated the code and tested it. I also tested if the URLs are generated correctly in the emails – they are. So I think you can do a final review now @stancl

Reply:
@stancl could you please take a look at the TenancyServiceProvider stub again now that it's updated? 
(the code example and the `overrideUrlInTenantContext` method + its call) https://github.com/archtechx/tenancy/pull/1044/files#diff-6ca30424f3bab5bd30feedc72b4d4e930244bc8f73fc6828396beb574df18fd0